src/shared/views/CometChatBadge/CometChatBadge.tsx

```diff
@@ -16,15 +16,11 @@ import styles from './styles';
  *
  */
 interface CometChatBadgeProps {
-  count: number;
+  count?: number;
   style?: BadgeStyle;
 }
 
-export const CometChatBadge = (props: CometChatBadgeProps) => {
-  const {
-    count= 0,
-    style: propsStyle= new BadgeStyle({}),
-  } = props;
+export const CometChatBadge = ({ count = 0, style = new BadgeStyle({}) }: CometChatBadgeProps) => {
```

Comment:
Technically, I made this change first. You don't need the `props` object for anything else, so just destructure inline. Plus, the correct spacing usage for the `=`.